### ISSUE TITLE
HID remapping

### ISSUE THREAD

**Issue by tomByrer** [hrvach/deskhop#84]
Just an idea; offer key & mouse remapping.  Handy for missing/reprogramming [media keys](https://github.com/hrvach/deskhop/issues/61), foreign characters, macros, etc

https://github.com/jfedor2/hid-remapper/

**Comment by jalmeroth** [comment]
I had the same thought, when taking screenlock away from @hrvach in my PR yesterday. 🫣

**Comment by hrvach** [comment]
This is a good idea. I'm currently trying to focus primarily on getting the basic features working for most people - multiple desktops, weird keyboards, non-working media keys etc. Once that's done, hopefully more fun stuff can be implemented :)

**Comment by tomByrer** [comment]
Yes, lower priority, though I wonder if adding this could help fix the "non-working media keys", or perhaps that code could give a clue.
eg: https://github.com/jfedor2/hid-remapper/issues/120#issuecomment-2004787215

**Comment by hrvach** [comment]
It definitely could, I'm going to order a keyboard with some media keys to finally have some hardware to test/develop on. The expectation is that this would help me figure it out and provide support for your plays, pauses and volumes :)

**Comment by chri2** [comment]
That's great news! I already was about to build a hid-remapper to get the keys back ;-). If you'd really only need it for testing I could send you one of [these](https://www.ebay.de/itm/312560384801). The media keys should be the same as the ones I asked about. Let me know if that'd help.

**Comment by 222Phoenix** [comment]
HID-remapper works with Deskhop, if anyone has both
